core/iwasm/common/wasm_runtime_common.h

```diff
@@ -49,6 +49,7 @@ extern "C" {
 #define GET_I64_FROM_ADDR(addr) (*(int64 *)(addr))
 #define GET_F64_FROM_ADDR(addr) (*(float64 *)(addr))
 #define GET_REF_FROM_ADDR(addr) (*(void **)(addr))
+#define GET_V128_FROM_ADDR(addr) (*(V128 *)(addr))
```

Comment:
Do you think we should define the macro below when WASM_CPU_SUPPORTS_UNALIGNED_ADDR_ACCESS macro is 0?

Reply:
right, updated.